### PR TITLE
Seed data

### PR DESCRIPTION
**Pull request by alexwolfmsft**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/database/seed-database-data.md](https://github.com/dotnet/docs-aspire/blob/23cb8aea093e6df014610b950e46468b1a52f1ff/docs/database/seed-database-data.md) | [Seed data in a database using .NET Aspire](https://review.learn.microsoft.com/en-us/dotnet/aspire/database/seed-database-data?branch=pr-en-us-407) |

### CODE REVIEW

File: docs/database/seed-database-data.md
Comment (IEvangelist) [inline docs/database/seed-database-data.md:1]: We're going to need a TOC entry, probably under the existing `Database` node.

**Review by IEvangelist — APPROVED** [review]
This looks good, thank you. We'll :shipit: when you make the suggested changes.

**Comment by IEvangelist** [comment]
Do you think that this PR covers this issue #64?

File: docs/database/seed-database-data.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+- **Source**: The source path of the volume mount, which is the physical location on your host.
+- **Target**: The target path in the container of the data you want to persist.
+- **VolumeMountType**: There are two types of volume mounts available:
+  - **Named mounts**: Storage managed by docker that your containers can use to persist data.
+  - **Bind mounts**: Files mounted from your host machine onto your container. Bind mounts are the approach used to seed data, since they allows you to map a script in your local app to the startup entry point in your container.
+
+Consider the following volume configuration code from a _Program.cs_ file in a sample **AppHost** project:
+
+```csharp
+var sql = builder.AddPostgresContainer("sql", sqlpassword)
```
[the remark this answers is not in the document]
Comment (alexwolfmsft) [reply docs/database/seed-database-data.md]: Fixed postgres example code

**Comment by alexwolfmsft** [comment]
> Do you think that this PR covers this issue #64?

I think it partially answers it - it sounds like in #64 they are trying to do traditional "Design time" migrations where they can just run migrations against their database or container using the ef core tools regardless of whether the app is running. However, the database container either doesn't exist or isn't accessible from local ef core tooling, which is why the example runs them during startup. I think this new doc shows them an alternate way of doing this, which they might be interested in - I don't think .NET Aspire supports traditional design time migrations at this point.

**Pull request merged**